server/src-lib/Hasura/Server/App.hs

```diff
@@ -530,9 +531,12 @@ httpApp corsCfg serverCtx enableConsole consoleAssetsDir enableTelemetry = do
     -- Health check endpoint
     Spock.get "healthz" $ do
       sc <- liftIO $ getSCFromRef $ scCacheRef serverCtx
-      if null $ scInconsistentObjs sc
-        then Spock.setStatus HTTP.status200 >> Spock.lazyBytes "OK"
-        else Spock.setStatus HTTP.status500 >> Spock.lazyBytes "ERROR"
+      dbOk <- checkDbConnection
+      if dbOk
```

Comment:
There was no content-type set when using `lazyBytes`, also no point using lazyBytes for such small strings.